src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  sourceString.split(';').forEach((style) => {
+    const [property, value] = style.split(':').map((part) => part.trim());
```

Comment:
Try to create constants, chaining is not a good idea for good readability

Reply:
Created)